### PR TITLE
Added region detectioin for change set if using arn

### PR DESCRIPTION
**Pull request by rayoogmy**
Fixes https://github.com/KCOM-Enterprise/cfn-square/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,158 +1,248 @@
...
+                        ] }
+           }
+
+<a name="SpecialRequirements"></a>	
+#### Special Requirements
+
+
+<a name="Author"></a>	
+#### Original Author
+Jonathan Rudge (jonathan.rudge@kcom.com)
```
Comment (steveparker-1984) [inline README.md]: Revert all change to this file

**Review by steveparker-1984 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rayoogmy) [reply README.md]: Did not realise that got in! Will revert.

**Review by steveparker-1984 — CHANGES_REQUESTED** [review]
LGTM. Need to increment version number though or it wont deploy

**Comment by rayoogmy** [comment]
Updated and squashed. Ready to merge.

**Review by steveparker-1984 — APPROVED** [review]

**Pull request merged**